src/workerd/server/workerd.capnp

```diff
@@ -643,6 +643,10 @@ struct Worker {
   tails @14 :List(ServiceDesignator);
   # List of tail worker services that should receive tail events for this worker.
   # See: https://developers.cloudflare.com/workers/observability/logs/tail-workers/
+
+  streamingTails @15 :List(ServiceDesignator);
+  # List of streaming tail worker services that should receive tail events for this worker.
+  # NOTE: This will be deleted in a future refactor, do not depend on this.
```

Comment:
Keep in mind that deleting stuff from these schemas can absolutely be a breaking change. Most likely this field would need to become obsoleted rather than deleted.

Reply:
Yeah I know, but that's the best we've got atm to get some experimental local dev working.